src/Microsoft.DotNet.Cli.Utils/IEnumerableExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.DotNet.Cli.Utils
```

Comment:
I made my class name match yours, so that who merges later will have to resolve a conflict with the name.